src/screens/Settings/components/OtherSettings/index.test.tsx

```diff
@@ -32,6 +32,7 @@ interface Props {
   toggleDiscordRPC: () => void
   useGameMode: boolean
   targetExe: string
+  isNativeMac: boolean
```

Comment:
sometimes you call it "isMacNative" here you call it "isNativeMac". Is this on porpuse ?